.github/workflows/delivery-ubuntu.yml

```diff
@@ -9,35 +9,33 @@ on:
       tag_name:
         description: The release tag to distribute
         required: true
-      ubuntu_ppa:
-        description: flag indicating this is an ubuntu release
-        required: true
-
 
 jobs:
     create-ppa:
-      #if: github.event_name == 'release'
       strategy:
         fail-fast: false
         matrix:
-          target: [xenial, bionic, focal]
           include:
-            - target: xenial
-              runner: ubuntu-16.04
-              image: docker://ubuntu:16.04
-              i386-go-version: 1.13
             - target: bionic
-              runner: ubuntu-18.04
-              image: docker://ubuntu:18.04
-              i386-go-version: 1.15
+              go-dep-entry: golang (>=1.16)
```

Comment:
It seems like this is the same all 3 times, why do we need this to be part of the matrix?

Reply:
You are right in that they are the same at the moment. There are two reasons why I left it.

1. If we want to fix xenial, it would be a different entry (something like golang-1.13).
2. As OS versions get older, the PPA packages may move from golang to golang-{{version}} based on the PPA maintainers discretion.

Happy to remove it from the matrix until it's necessary if you think it causes confusion.

Reply:
Simplified the pipeline which simplifies this as well. Please re-review.